mailbox/store/src/main/java/org/apache/james/mailbox/store/json/event/dto/MailboxSessionDataTransferObject.java

```diff
@@ -1,111 +0,0 @@
-/****************************************************************
```

Comment:
LocaleDataTransferObject class can also be removed

Reply:
addressed in part1 https://github.com/linagora/james-project/pull/1980/commits/eca779fca8132922b815da0d326609f811adceca